contribute.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "DXR",
+    "description": "An intelligent source code browser.",
+    "repository": {
+        "url": "https://github.com/mozilla/dxr",
+        "license": "MIT",
+        "tests": "https://ci.mozilla.org/job/dxr/"
+    },
+    "participate": {
+        "home": "https://dxr.readthedocs.org/en/latest/community.html",
+        "docs": "https://dxr.readthedocs.org/en/latest/development.html",
```

Comment:
Air Mozilla's points to the part of their docs for setting up a developer instance, so I've been going with that. I'm not sure the semantics are super-clear.